platforms/core-configuration/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/accessors/Stage1BlocksAccessorClassPath.kt

```diff
@@ -155,7 +156,7 @@ abstract class AbstractStage1BlockAccessorsUnitOfWork(
     }
 
     override fun identify(identityInputs: MutableMap<String, ValueSnapshot>, identityFileInputs: MutableMap<String, CurrentFileCollectionFingerprint>) =
-        UnitOfWork.Identity { "$classLoaderHash-$identitySuffix" }
+        Identity { "$classLoaderHash-$identitySuffix" }
```

Comment:
Just a minor note but I did find `UnitOfWork.Identity` more expressive than just `Identity` since that's quite an overloaded term.